channeldb/db.go

```diff
@@ -240,6 +267,61 @@ func Open(dbPath string, modifiers ...OptionModifier) (*DB, error) {
 	return db, err
 }
 
+func resetChanStateCache(cache *kvdb.Cache) error {
+	cache.Wipe()
+
+	cache.SkipBucket(closedChannelBucket)
+	cache.SkipBucket(closeSummaryBucket)
+	cache.SkipBucket(fwdPackagesKey)
+	cache.SkipBucket(revocationLogBucket)
+
+	if err := cache.AddTopLevelBucket(nodeBucket); err != nil {
+		return err
+	}
+
+	if err := cache.AddTopLevelBucket(edgeBucket); err != nil {
+		return err
+	}
+
+	if err := cache.AddTopLevelBucket(edgeIndexBucket); err != nil {
+		return err
+	}
+
+	if err := cache.AddTopLevelBucket(graphMetaBucket); err != nil {
+		return err
+	}
+
+	// Add other buckets holding channel state to the channeldb cache.
+	if err := cache.AddTopLevelBucket(openChannelBucket); err != nil {
+		return err
+	}
+
+	// Won't fetch.
+	if err := cache.AddTopLevelBucket(closedChannelBucket); err != nil {
+		return err
+	}
+
+	// Won't fetch.
+	if err := cache.AddTopLevelBucket(closeSummaryBucket); err != nil {
+		return err
+	}
+
+	// Won't fetch.
+	if err := cache.AddTopLevelBucket(fwdPackagesKey); err != nil {
+		return err
+	}
+
+	if err := cache.AddTopLevelBucket(outpointBucket); err != nil {
```

Comment:
It shouldn't grow very fast at least since we only store past and current channel outpoints there. We can certainly skip it but I would not expect it to cause much issues.